lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -691,11 +691,12 @@ describe('AlfrescoViewerComponent', () => {
         });
 
         it('should render close viewer button if it is not a shared link', (done) => {
+            component.closeButtonPosition = CloseButtonPosition.Left;
             fixture.detectChanges();
             fixture.whenStable().then(() => {
                 fixture.detectChanges();
-                expect(element.querySelector('[data-automation-id="adf-toolbar-back"]')).toBeDefined();
-                expect(element.querySelector('[data-automation-id="adf-toolbar-back"]')).not.toBeNull();
+                expect(element.querySelector('[data-automation-id="adf-toolbar-left-back"]')).toBeDefined();
```

Comment:
One of those statements is obsolete, first you check if the element is defined and then if it's not a null

Reply:
Removed obsolete statements